packages/@atjson/source-html/src/converter/headings.ts

```diff
@@ -0,0 +1,57 @@
+import Document, { is, AnnotationConstructor } from "@atjson/document";
+import { Heading } from "@atjson/offset-annotations";
+import { parseCSS, toAlignment } from "./utils";
+import { H1, H2, H3, H4, H5, H6 } from "../annotations";
+
+function convert(
+  doc: Document,
+  type: AnnotationConstructor<any, any>,
+  level: 1 | 2 | 3 | 4 | 5 | 6
+) {
+  doc
+    .where((annotation) => is(annotation, type))
+    .as("heading")
+    .outerJoin(
+      doc
+        .where((annotation) => annotation.attributes.dir != null)
+        .as("directions"),
+      (heading, direction) =>
+        heading.start >= direction.start && heading.end <= direction.end
```

Comment:
would you mind changing this so that we only use less than signs? I think Bach has that code formatting preference

Reply:
I can change this, but I've never been aware of this— I'm happy if we can standardize this. @bachbui could you elaborate on this style preference?

Reply:
My preference for this is specifically around text ranges: we often want to query for things that are "inside" other things, and I find it easier to see at a glance when the queries look like
```
outside.start <= inside.start && inside.end <= outside.end
```

For other things I don't have a strong preference (`Array.indexOf(a) > -1` is totally fine)

Reply:
Would you like me to change this?

Reply:
Yes plz